contrib/license-header.txt

```diff
@@ -0,0 +1,14 @@
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
```

Comment:
Can you create a documentation file (something like `contrib/add-license-headers.md`) with the command you used to run?  It would be helpful to have the command so we can run this periodically.

Reply:
Added in 14c97f6